.github/workflows/lint.yml

```
@@ -0,0 +1,49 @@
+name: Lint
+
+# Github actions for Turborepos
+# ref: https://turborepo.com/docs/guides/ci-vendors/github-actions
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    types: ["opened", "synchronized"]
```

actionlint (1.7.4)

invalid activity type "synchronized" for "pull_request" Webhook event. available types are "assigned", "auto_merge_disabled", "auto_merge_enabled", "closed", "converted_to_draft", "demilestoned", "dequeued", "edited", "enqueued", "labeled", "locked", "milestoned", "opened", "ready_for_review", "reopened", "review_request_removed", "review_requested", "synchronize", "unassigned", "unlabeled", "unlocked"

(events)